src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,86 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your alert coverage by implementing the following recommendations. 
+
+Read on to learn the best practices for:
```

Comment:
In general, I'd love to see the section headers in this doc get more descriptive. For example, instead of "Policies" something like "Get the most out of your policies" or even just "Organize your policies". Get some verbs in there to make it easier to skim those section headers.

Reply:
Once you've rewritten the section headers, I think line 21 could be rewritten. Something like.... er.... "Use these links to jump to the topic you're interested in learning more about" or something... 

It's also possible that you could lose this whole links section (it's in the right nav) once you've rewritten those section headers. Something to think about :)